File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/Elevation.java
package com.example.testlunu.net.acu.cityinfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Elevation {

    @SerializedName("Metric")
    @Expose
    public Metric metric;
    @SerializedName("Imperial")
    @Expose
    public Imperial imperial;

}

File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/Elevation.kt
package com.example.testlunu.net.acu.cityinfo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Elevation (
    @SerializedName("Metric")
    @Expose
    var metric: Metric?,
    @SerializedName("Imperial")
    @Expose
    var imperial: Imperial?
)